### PR TITLE
Update link to API viewer

### LINKED ISSUE #2: Swagger Viewer points to 404
**Link to problem area**: <https://spec.matrix.org/latest/#matrix-apis>

**Issue**

The link on »The Matrix Client-Server API Swagger Viewer is useful for browsing the Client-Server API.« points to <https://matrix.org/docs/api/client-server/>, but this gives an »404 Not Found«.

**Expected behaviour**

Either remove the link or point to where the resource is.

### PR DESCRIPTION
**Pull request by afranke**
Fixes #2

Preview: https://pr1308--matrix-spec-previews.netlify.app

### CODE REVIEW

**Comment by afranke** [comment]
CI seems to fail on the lack of changelog item, which shouldn’t apply for this type of change.

**Comment by turt2live** [comment]
We have an "internal" changelog for this sort of stuff these days - if one could be added, it would be appreciated.

**Review by richvdh — COMMENTED** [review]
lgtm other than the lack of changelog, which I will fix

File: content/_index.md [hunk 1]
```diff
@@ -31,8 +31,8 @@ Additionally, this introduction page contains the key baseline
 information required to understand the specific APIs, including the
 section the [overall architecture](#architecture).
 
-The [Matrix Client-Server API Swagger
-Viewer](https://matrix.org/docs/api/client-server/) is useful for
+The [Matrix API
+Viewer](https://matrix.org/docs/api/) is useful for
```
Comment (richvdh) [inline content/_index.md:35]:
```suggestion
The [Matrix API Viewer](https://matrix.org/docs/api/) is useful for
```

**Review by richvdh — APPROVED** [review]

**Pull request merged**